python/idsse_common/idsse/common/sci/vectaster.py

```diff
@@ -368,9 +374,13 @@ def _pixels_for_linestring(
         list[Pixel]: List of x,y tuples in pixel space
     """
     coords = linestring.coords
-    pixels = _pixels_for_line_seg(coords[0], coords[1])
+    pixels = set(_pixels_for_line_seg(coords[0], coords[1]))
```

Comment:
Just to check for understanding: it looks like we turn the pixel list into a set to prevent duplicates here, but the `_pixels_for_line_seg()` function we called also turns them into a set before returning them to us. 

Is there a reason that `_pixels_for_line_seg()` isn't guaranteed to return unique pixels, but the similar function `_pixels_for_polygon()` is? Maybe because Polygons have boundaries + edges so the logic in that function is more complex and complete?